sql-statements/sql-statement-select.md

```diff
@@ -31,7 +31,24 @@ SelectStmtOpts ::=
     SelectStmtStraightJoin
 
```

Comment:
Please report an issue with:
```
SelectStmtBasic ::=
    "SELECT" SelectStmtOpts Field ("," Field)* ( "HAVING" Expression)?
```
since `SELECT * HAVING 1` does not parse.

Also in parser.y, SelectStmtBasic is the first part of SelectStmtFromTable...
But I assume this can all be addressed in a separate issue.

Most likely SelectStmtBasic should not have HavingClause?